java/java.lsp.server/vscode/package.json

```diff
@@ -211,6 +211,11 @@
 				"title": "Compile Workspace",
 				"category": "Java"
 			},
+			{
+				"command": "java.workspace.new",
+				"title": "New from Template...",
+				"category": "Java"
+			},
```

Comment:
Yeah a keybinding would be useful. 

Also I think it would be nice to have a keybinding without having to bring the file explorer into focus. 

Say for example I am editing a Java file and I create a new test from a template for it, it should create the file in the same package but in the `src/test/java` tree instead.

Reply:
Choose a key binding combination and I'll define it!

Good idea to expect creation of a new file as a sibling of currently active editor. Done in 9d9f851, but (since 819c639) we still ask the user to confirm the correct path in such case.

Reply:
Creation of unit tests is a completely different use-case and deserves different action (possibly) and certainly a different PR.